data/remnant/remnant.txt

```diff
@@ -475,7 +500,109 @@ phrase "friendly remnant"
 		"."
 		"!"
 
-phrase "friendly remnant"
+phrase "remnant no scan"
+	word
+		"Please"
+	word 
+		" "
+	word
+		"do not"
+		"try not to"
+	word
+		" "
+	word
+		"scan"
+	word
+		" "
+	word
+		"us."
+		"our ships."
+
+phrase "remnant no scan"
+	word
+		"Careless scans costs lives."
+		"Unasked for scans are unwelcome."
+		"Scanning is a privilege to be earned."
+
+phrase "remnant scan"
+	word
+		"It would be appreciated if"
+		"It would be helpful if"
+	word
+		" "
+	word
+		"you"
+	word
+		" "
+	word
+		"scanned"
+		"took a look at"
+		"kept an eye on"
+		"checked"
+	word
+		" "
+	word
+		"my ship."
+		"our ships."
+		"everything in the Ember Waste."
+
+phrase "remnant scan"
+	word
+		"Doing a"
+		"Performing a"
+		"Carrying out a"
+	word
+		" "
+	word
+		"sensor sweep"
+		"sweep"
+		"check"
+		"scan"
+	word
+		" "
+	word
+		"for"
+	word
+		" "
+	word
+		"hitchhikers"
+		"clingers"
+		"unidentified objects"
+		"security breaches"
+		"hostile entities"
+	word
+		" "
+	word
+		"on my ship"
+		"on our ships"
+		"on all ships"
+	word
+		" "
+	word
+		"is"
+		"can be"
+	word
+		" "
+	word
+		"quite helpful."
+		"very useful."
+		"essential for security."
+		"much appreciated."
+		"a good precaution."
+
+phrase "remnant secrecy"
+	word
+		"Loose lips sink ships."
+		"Careless talk costs lives."
+		"Keep mum."
```

Comment:
```suggestion
```
This is British slang - I've actually never heard it used. If you're strongly attached to it, it's fine, but I'd prefer we take it out.

Reply:
I like it, given that it breaks the Ameri-centrism in the linguistics. Gives it a different feel from the rest of humanity.

Reply:
Fair enough!